signac/contrib/job.py

```diff
@@ -240,9 +248,10 @@ def update_statepoint(self, update, overwrite=False):
         update : dict
             A mapping used for the state point update.
         overwrite :
-            Set to true, to ignore whether this update overwrites parameters,
-            which are currently part of the job's state point.
-            Use with caution! (Default value = False)
+            Set to true to ignore whether this update overwrites parameters
+            that are currently part of the job's state point.
+            If true, equivalent to ``job.reset_statepoint()``. Use with caution!
+            (Default value = False)
```

Comment:
```suggestion
            If True, this method will ignore parameters that are currently
            part of the job's state point. This is equivalent to
            :meth:`~.reset_statepoint`. Use with caution! (Default value =
            False)
```

Reply:
I like your rewording, but it gave me an idea on how to make it even more concise and better emphasize the relationship with `reset_statepoint`.